NEWS.md

```diff
@@ -532,7 +532,7 @@ https://github.com/ruby/lrama/pull/62
 
 ### Runtime configuration for error recovery
 
-Make error recovery function configurable on runtime by two new macros.
+Meke error recovery function configurable on runtime by two new macros.
```

Comment:
Is this your intended change?

Reply:
Negative; I don't know how that got in there 😅 

Fixed!